config/PolyDbImplCfg.hpp

```diff
@@ -10,9 +10,9 @@
 
 namespace {
 
-    enum {
-        POLYDB_NUM_DB_ENTRIES = 25
-    };
+    // enum {
```

Comment:
If the entry count is inferred from the number of entries in the enum, then delete this whole file.